Mathlib/SetTheory/Game/PGame.lean

```diff
@@ -1267,6 +1488,30 @@ instance : Add PGame.{u} :=
     · exact fun i => IHxr i y
     · exact IHyr⟩
 
+@[simp]
+lemma leftMoves_mk_add {xl xr yl yr} {xL xR yL yR} :
```

Comment:
On the one hand, I'm surprised we didn't have these. It might be worth it adding the versions for multiplication too, if we're missing them. 

On the other hand, the intended way to talk about moves of addition, etc. is though `toLeftMovesAdd`, etc. Unfolding things in terms of `mk` like this should be a last resort. Are these lemmas getting used in this PR?

Reply:
Yeah, the `mk` lemmas are useful when you need definitional equalities. That said, we should avoid them outside of basic API development like you mention.